### PR TITLE
[RDY] Don't store translated mtype description.

### PR DESCRIPTION
**Pull request by AMurkin**
As proposed by @codemime in #21645

### CODE REVIEW

File: src/monster.cpp [hunk 1]
```diff
@@ -429,7 +429,7 @@ int monster::print_info(WINDOW* w, int vStart, int vLines, int column) const
     const auto hp_desc = hp_description( hp, type->hp );
     mvwprintz( w, vStart++, column, hp_desc.second, "%s", hp_desc.first.c_str() );
 
-    std::vector<std::string> lines = foldstring(type->description, getmaxx(w) - 1 - column);
+    std::vector<std::string> lines = foldstring( type->get_description().c_str(), getmaxx(w) - 1 - column );
```
Comment (BevapDin) [inline src/monster.cpp]: Here you don't need to call `c_str`, `foldstring` works with an `std::string`.

`description` is also used (without translation) in iuse.cpp - you should probably make it private in `mtype`, so your compiler will tell where it's used directly.

Comment (AMurkin) [inline src/monster.cpp]: Thank you for your advice.

**Comment by codemime** [comment]
Though the class is "wide open", it would make sense to put [the member variable](https://github.com/CleverRaven/Cataclysm-DDA/pull/21651/files#diff-e193e14055629a16377656c4d5f1edb0L200) into the `private` section. This might also highlight some more occurrences of the variable.

**Comment by AMurkin** [comment]
@codemime There is no more. I've test it locally, but not commited that change. I did it now.

**Comment by AMurkin** [comment]
I got it. `writeFile("catabindings.cpp", cpp_output)` in generate_bindings.lua.

**Comment by BevapDin** [comment]
Remove the line (ca line 1807) with the description from the lua/class_definitions.lua to fix the build.

**Comment by AMurkin** [comment]
@BevapDin Could it cause problems for lua-mods?

**Pull request merged**